src/integrations/weatheralerts.ts

```diff
@@ -75,7 +75,8 @@ export default class Weatheralerts implements MeteoalarmIntegration {
 			'Tropical Storm': MeteoalarmEventType.Hurricane,
 			'Hurricane': MeteoalarmEventType.Hurricane,
 			'Air Quality': MeteoalarmEventType.AirQuality,
-			'Rip Current': MeteoalarmEventType.CoastalEvent // https://github.com/MrBartusek/MeteoalarmCard/issues/183
+			'Rip Current': MeteoalarmEventType.CoastalEvent, // https://github.com/MrBartusek/MeteoalarmCard/issues/183
+			'Special Weather': MeteoalarmEventType.SpecialWeather
```

Comment:
Hey,  for the `Special Weather Statement` the best fit would be `MeteoalarmEventType.Unknown` since that is what it was. It can really be anything and these warnings are not limited to thunderstorms.

Reply:
Roger, I’ll make these changes and push up shortly. Thank you